context.go

```diff
@@ -36,8 +36,16 @@ func (c *Context) Next() {
 	}
 }
 
-func (c *Context) Request() *fasthttp.Request   { return &c.fc.Request }
-func (c *Context) Response() *fasthttp.Response { return &c.fc.Response }
+func (c *Context) RequestMethodIsOptions() bool { return c.fc.Request.Header.IsOptions() }
```

Comment:
Removing RequestMethod would be intuitive

Reply:
done